src/App/InformasjonOmTilgangsstyringSide/InformasjonOmTilgangsstyringSide.tsx

```diff
@@ -89,48 +132,30 @@ const InformasjonOmTilgangsstyringSide = () => {
                         </Lenke>
                     </Ekspanderbartpanel>
 
-                    <Ekspanderbartpanel tittel="Tilskuddsbrev for NAV-tiltak" border>
+                    <Ekspanderbartpanel tittel="Innmelding for lønnskompensasjon og refusjon" border>
                         <Normaltekst>
-                            For å få tilgang til digitale tilskuddsbrev om NAV-tiltak må du ha en av
-                            disse lederrollene i Altinn:
+                            Rollene er de samme som kreves for å sende inntektsmelding. Du må dermed
+                            ha disse tilgangene (rollene) på vegne av den aktuelle virksomheten:
                         </Normaltekst>
                         <ul>
-                            <li>daglig leder/administrerende direktør</li>
-                            <li>styrets leder</li>
-                            <li>innehaver</li>
-                            <li>komplementar</li>
-                            <li>deltaker med delt ansvar</li>
-                            <li>deltaker med fullt ansvar</li>
-                            <li>bestyrende reder</li>
-                            <li>norsk representant for utenlandsk enhet</li>
-                            <li>bostyrer</li>
+                            <li>ansvarlig revisor</li>
+                            <li>lønn og personalmedarbeider</li>
+                            <li>regnskapsfører lønn</li>
+                            <li>regnskapsfører med signeringsrettighet</li>
+                            <li>regnskapsfører uten signeringsrettighet</li>
+                            <li>revisormedarbeider</li>
+                            <li>kontaktperson NUF</li>
```

Comment:
kontaktperson nuf kan skrives til norsk representant for utenlandsk enhet for å være konsekvent

Reply:
Fikset